src/test/testSchemaGenerator.ts

```diff
@@ -907,6 +911,86 @@ describe('providing useful errors from resolve functions', () => {
     });
   });
 
+  it('will use default resolver of non found when asserting undefined', () => {
```

Comment:
basiclly to cover this flow:
https://github.com/apollostack/graphql-tools/blob/master/src/schemaGenerator.ts#L416

I'm returning from thread's object of args.
and there is no resolver for Thread, so it expect to use defaultResolveFn